nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -445,6 +493,31 @@ in {
         "${pkgs.libfido2}/lib/libfido2.so.1"
       ] ++ jobScripts;
 
+      users = {
+        root = {};
```

Comment:
Wouldn't it be a good idea to force UID 0 here? Just in case

Reply:
It pulls uid from the global config by default, so it should already be set to 0 from [`misc/ids.nix`](https://github.com/NixOS/nixpkgs/blob/3fb8eedc450286d5092e4953118212fa21091b3b/nixos/modules/misc/ids.nix#L42) and [`config/users-groups.nix`](https://github.com/NixOS/nixpkgs/blob/3fb8eedc450286d5092e4953118212fa21091b3b/nixos/modules/config/users-groups.nix#L548), right?

Reply:
Yea it'll get it from stage 2, and I'm comfortable leaving that as is. If that ever gets messed up, we're in for all sorts of hurt anyway.